living-doc-mops-streaming-opcons.md

```diff
@@ -0,0 +1,211 @@
+- Common media client data (CMCD): initial findings (<https://dl.acm.org/doi/10.1145/3458306.3461444>)
+- SDNDASH: Improving QoE of HTTP Adaptive Streaming Using Software Defined Networking (<https://dl.acm.org/doi/10.1145/2964284.2964332>)
+- Caching in HTTP Adaptive Streaming: Friend or Foe? (<https://dl.acm.org/doi/10.1145/2578260.2578270>)
+- A Survey on Multi-Access Edge Computing Applied to Video Streaming: Some Research Issues and Challenges
+(<https://ieeexplore.ieee.org/document/9374553>)
+- The Ultimate Guide to Internet Congestion Control (<https://www.compiralabs.com/ultimate-guide-congestion-control>)
+
+## QoE Metrics
+
+The following papers describe various QoE metrics one can use in streaming applications.
+
+- QoE Management of Multimedia Streaming Services in Future Networks: a Tutorial and Survey (<https://ieeexplore.ieee.org/document/8930519>)
+- A Survey on Quality of Experience of HTTP Adaptive Streaming (<https://ieeexplore.ieee.org/document/6913491>)
+- QoE Modeling for HTTP Adaptive Video Streaming–A Survey and Open Challenges (<https://ieeexplore.ieee.org/document/8666971>)
+
+## Point Clouds and Immersive Media
+
+The following papers explain the latest developments in the immersive media domain (for video and audio) and the developing standards for such media.
+
+- A Survey on Adaptive 360° Video Streaming: Solutions, Challenges and Opportunities (<https://ieeexplore.ieee.org/document/9133103>)
+- MPEG Immersive Video Coding Standard (<https://ieeexplore.ieee.org/document/9374648>)
+- Emerging MPEG Standards for Point Cloud Compression (<https://ieeexplore.ieee.org/document/8571288>)
+- Compression of Sparse and Dense Dynamic Point Clouds—Methods and Standards (<https://ieeexplore.ieee.org/document/9457097>)
+- MPEG Standards for Compressed Representation of Immersive Audio (<https://ieeexplore.ieee.org/document/9444109>)
+- An Overview of Omnidirectional MediA Format (OMAF) (<https://ieeexplore.ieee.org/document/9380215>)
+- From Capturing to Rendering: Volumetric Media Delivery with Six Degrees of Freedom (<https://ieeexplore.ieee.org/document/9247522>)
+
+# Open-Source Tools
+
+- 5G-MA: <https://www.5g-mag.com/reference-tools>
+- dash.js: <https://reference.dashif.org/dash.js/latest/samples/>
+- DASH-IF Conformance: <https://conformance.dashif.org>
+- ExoPlayer: <https://github.com/google/ExoPlayer>
+- FFmpeg: <https://www.ffmpeg.org/>
+- GPAC: <https://gpac.wp.imt.fr/>
+- hls.js: <https://github.com/video-dev/hls.js>
+- OBS Studio: <https://obsproject.com/>
+- Shaka Player: <https://github.com/google/shaka-player>
+- Shaka Packager: <https://github.com/google/shaka-packager>
+- Traffic Control CDN: <https://trafficcontrol.apache.org/>
+- VideoLAN: <https://www.videolan.org/projects/>
+- video.js: <https://github.com/videojs/video.js>
+
+# Technical Events
+
+- ACM Mile High Video (MHV): <https://mile-high.video/>
```

Comment:
ACM is using the name "ACM Mile-High Video (MHV)" , so there is a dash between mile and high.